nncf/quantization/algorithms/min_max/torch_backend.py

```diff
@@ -315,6 +319,10 @@ def get_ignored_metatypes(model_type: ModelType, device: TargetDevice) -> List[O
                 om.PTDivMetatype,
                 om.PTMaxMetatype,
                 om.PTSqueezeMetatype,
+                om.PTLayerNormMetatype,
+                om.PTModuleLayerNormMetatype,
+                om.PTGroupNormMetatype,
+                om.PTModuleGroupNormMetatype,
```

Comment:
Test were missed it this pr https://github.com/openvinotoolkit/nncf/pull/1975 thus ignored metatypes solver logic is not tested now.
CC: @l-bat